packages/generate-arg-types/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@webstudio-is/generate-arg-types",
+  "version": "0.0.1",
+  "description": "Generate components property types",
+  "author": "Webstudio <github@webstudio.is>",
+  "homepage": "https://webstudio.is",
+  "type": "module",
+  "scripts": {
+    "typecheck": "tsc --noEmit",
+    "checks": "yarn typecheck && yarn lint",
+    "build": "rm -rf .bin tsconfig.tsbuildinfo && tsc && chmod +x .bin/index.js",
```

Comment:
I would not build .bin at all. We have tsx loader which does all work.
NODE_OPTIONS='--loader=tsx' svgo-jsx svgo-jsx.config.ts